### DISCUSSION TITLE
Touchscreen is not working

### DISCUSSION THREAD

**Discussion by FoolProcessor** [shanteacontrols/OpenDeck#263] [Q&A]
Hey guys,
I have a problem setting up my touchscreen.
Hardware: OpenDeck Board L, Nextion intelligent series NX1060P101
I followed the instructions on the wiki but couldn't get it to work.
In Nextion Editor I set up a picture on page 0 and a hotspot (sized to the whole screen as I just wanted to test the feature). I set up another page with an "on" state picture. In Debug mode  the hotspot sends the right touch press event.
So I uploaded it to my touchscreen and went inside OpenDeck configurator ( it is the first time I use this tool) . I enabled touchscreen, clicked on table and put the values from Nextion editor in. 
But what to do now? The Screen doesn't react to pushing on it (it just turn to page 0 when i connect the OpenDeck board to my pc). 
I went to "Button" -> "touchscreen" and edited button 192. Nothing happens either.
Sometimes when I go back to touchscreen tab the touchscreen is disabled again.
Where do I fail here? I hope its just user error.
Also the activity window shows a "Note On" message from 77-127 without me doing anything. Should it show this?
I hope you can help me. Thanks in Advance

![Nextion 1](https://github.com/shanteacontrols/OpenDeck/assets/155238969/cd155638-c628-4449-b158-c97906f2ab58)
![Nextion 2](https://github.com/shanteacontrols/OpenDeck/assets/155238969/2b92d260-3f85-40ce-9032-1ee9f6bfd02b)
![Nextion 3](https://github.com/shanteacontrols/OpenDeck/assets/155238969/257f64fe-8f84-4970-ac9b-67b447ffd2a5)
![Nextion 4](https://github.com/shanteacontrols/OpenDeck/assets/155238969/f0849515-8929-49d6-943d-4853ac9a6a50)
![OpenDeck Board](https://github.com/shanteacontrols/OpenDeck/assets/155238969/b441e78f-993b-44eb-9542-2f8116b58b75)

**Comment by paradajz** [comment]
Have you added the required code to the `Program.s`?

**Reply by FoolProcessor** [reply]
I pasted this in there from the tutorial post:
//The following code is only run once when power on, and is generally used for global variable definition and power on initialization data
int sys0=0,sys1=0,sys2=0     //At present, the definition of global variable only supports 4-byte signed integer (int), and other types of global quantity declaration are not supported. If you want to use string type, you can use variable control in the page to implement
bauds=38400
baud=38400
page 0                       //Power on start page 0

**Reply by paradajz** [reply]
Attach your design file.

**Reply by FoolProcessor** [reply]
HMI isn't supported. So i uploaded it on google drive:
https://drive.google.com/file/d/1cGtRvBvvhjpTdO6OflIgNnXorZcboC54/view?usp=sharing

**Reply by FoolProcessor** [reply]
@paradajz could you find out what the problem is?

**Comment by FoolProcessor** [comment]
Huge user error. I misinterpreted the tx mark on the Board as a tx channel. Connected the touchscreens tx to boards rx. Now it works.
